pkg/controller/types/group.go

```diff
@@ -0,0 +1,72 @@
+// Copyright 2021 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package types
+
+import (
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/apimachinery/pkg/types"
+
+	"github.com/vmware-tanzu/antrea/pkg/apis/controlplane"
+)
+
+type GroupReference struct {
+	// Namespace of the Group. It's empty for ClusterGroup.
+	Namespace string
```

Comment:
Why we need Namespace here if we support only ClusterGroup? Is it for internal groups generated for standalone selectors? But we already have Namespace in GroupSelector; could we just use that?

Reply:
this one is a bit forward looking, where in if/when we introduce Namespaced Groups CRD, we could insert the Namespace here. but will remove for now.

Reply:
Ok. I would support to remove it for now. I saw you already do that. But then the new question is do we still need a struct or just a Name string is good?

Reply:
replaced with `GroupReference` with `ClusterGroup`